projects/MailClient/MailClient/Dictionary/LogMessage.cs

```diff
@@ -1,9 +0,0 @@
-﻿namespace MailClient.Dictionary
-{
-    public static class LogMessage
-    {
-        public static string WrongEmailAdress = "You didn't enter email address correctly";
```

Comment:
jako że są to pewnego rodzaju stałe, warto je zaznaczyć jako `readonly` albo `const`